db/migrate/20170619161514_add_options_to_ext_management_system.rb

```diff
@@ -0,0 +1,34 @@
+class AddOptionsToExtManagementSystem < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+    serialize :options
+  end
+  class CustomAttribute < ActiveRecord::Base
+    serialize :serialized_value
+  end
+
+  def up
+    add_column :ext_management_systems, :options, :text
```

Comment:
I get that, I think the code should work the same except for dropping the serialize and some other code in the tests.
That way we get native JSON support (for example query only the `http_proxy` out of options) in the db layer.

@durandom wdyt?